package.json

```diff
@@ -4,7 +4,7 @@
    "author": "Deepak Sattiraju",
    "license": "MIT",
    "scripts": {
-      "build": "npx ncc build src/run.ts -o lib",
+      "build": "npm i ncc && ncc build src/run.ts -o lib",
```

Comment:
does this not need to be -g?

also unrelated do we want to change the author to AKS Atlanta

Reply:
yep, but i meant to still use npx, just use npm i without the --no-bin-links so that npx can find it

Reply:
that way we dont have to install globally and can use the one from the package.json still